docs/using-mina/how-to-use-zkapp.mdx

```diff
@@ -21,11 +15,7 @@ import Subhead from '@site/src/components/common/Subhead';
 1. Install a zkApp-compatible [Mina wallet](../using-mina/install-a-wallet).
 2. Make sure that your Mina wallet contains MINA to pay for transaction fees. 
 
-:::note
-
-The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps currently.
-
-:::
+The Mina community has created a variety of different wallets. Only the [Auro Wallet for Chrome](https://www.aurowallet.com) supports interactions with zkApps.
```

Comment:
see https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide#tone-and-content

> Write [timeless documentation](https://developers.google.com/style/timeless-documentation) that avoids words and phrases that anchor the documentation to a point in time or assume knowledge of prior or future products and features. Words to avoid: now, new, soon, not yet, latest, future, and currently.